assets/js/application-debug.js

```diff
@@ -26,8 +26,16 @@ const debug = (function(){
         if(log){
             const configDebug = application.object.config.debug;
             if(typeof configDebug === 'boolean'){
-                console.log(log)
-            
+                if(configDebug)console.log(log)
+            //}else if(configDebug.length){ // filter array
+            }else if( configDebug.constructor === Array){
+                //console.log('array')
+                for(let item in configDebug){
+                    
+                    if(log.indexOf(configDebug[item])>0){
```

Comment:
indexOf returns -1 if not found
indexOf returns 0 or larger if found. If in this line the line starts with the term the result is false while it should be true???
"snake".indexOf("sna") will return in 0 (zero). Got it?

Reply:
so might change to indexOf(...) >= 0 ?

Reply:
Absolutely get your point, but in this particular case the string which is searched for is never at position 0 (function:method is logged; method needs to be filtered, not function), changed it tough.